test/api/unit/services/SQS.test.js

```diff
@@ -101,6 +101,18 @@ describe("SQS", () => {
         }).catch(done)
       })
 
+      it("should respect the path component of a custom domain when setting BASEURL in the message", done => {
```

Comment:
It's not clear to me this new test covers all of the cases within `baseURLForCustomDomain` in `SQS.js`